zach_kaz/dlistsZK.py

```diff
@@ -0,0 +1,79 @@
+class Node:
+    def __init__(self,value,name):
+        self.value=value
+        self.name=name
+        self.next=None
+        self.previous=None
+
+class dList:
+    def __init__(self,value,name):
+        node=Node(value,name)
+        self.head=node
+    
+    def addNode(self,value,name):
+        node=Node(value,name)
+        runner=self.head
+        while (runner.next!=None):
+            runner=runner.next
+        runner.next=node
+        node.previous=runner
+    
+    def removeNode(self,finder):
+        x=self.head
+        if x.value==finder:
+            self.head=x.next
```

Comment:
also need to set the previous of second node to null
return after removing the first node